Tests/testProfile.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from server.profile import getProfile
+from server.matches import matchUser
+
+class TestProfile(unittest.TestCase):
+    def test_get_valid_profile(self):
+        userId = 11
+        exp_profile = {"response": "Success", "name": 'melo2', "age": 21,\
+                 "bio": 'sad', "gender": 'Female', "education": 'loo',\
+                      "interests": 'sad', "genderPreference": 'Female', "maxDistance": 10
+                      }
+        profile = getProfile(userId)
+        self.assertEqual(exp_profile, profile)
+
+    def test_get_invalid_profile(self):
+        userId = -1
+        exp_profile = {"response": "invalid userId"}
+        profile = getProfile(userId)
+        self.assertEqual(exp_profile, profile)
+
```

Comment:
Maybe you can try a userID that is not negative but out of range (ex. 1000) or other userID created without profile information.

Reply:
I think that would be testing the same as having a negative user id, so this test case should already be covered here.